packages/flutter/test/rendering/mouse_tracking_cursor_test.dart

```diff
@@ -17,7 +15,7 @@ import './mouse_tracking_test_utils.dart';
 
 typedef MethodCallHandler = Future<dynamic> Function(MethodCall call);
 
-TestMouseTrackerFlutterBinding _binding = TestMouseTrackerFlutterBinding();
+TestMouseTrackerFlutterBinding? _binding = TestMouseTrackerFlutterBinding();
```

Comment:
Why nullable? Looks like it never is null?

Reply:
On line 21 below there is a null check, that code is dead if this is non-nullable.

Reply:
That null check down there doesn't really make sense because it never can be null in the first place...

Reply:
That's what I thought.... thanks for the sanity check! I'll nix it!